src/wp-includes/user.php

```diff
@@ -1561,10 +1561,15 @@ function update_user_caches( $user ) {
  *
  * @since 3.0.0
  * @since 4.4.0 'clean_user_cache' action was added.
+ * @since 5.7.0 Refreshes the global user instance if cleaning the user cache for the logged in user.
```

Comment:
```suggestion
 * @since 5.8.0 Refreshes the global user instance if cleaning the user cache for the current user.
```

- Targeting 5.8 for this patch.
- May be more clear to state current user instead of logged in user. Why? The "current user" is programmatically changeable.